src/Drupal/Driver/Cores/Drupal8.php

```diff
@@ -385,7 +392,7 @@ public function getExtensionPathList() {
    * @param array $base_fields
    *   Base fields to be expanded in addition to user defined fields.
    */
-  public function expandEntityBaseFields($entity_type, \stdClass $entity, array $base_fields) {
+  public function expandEntityBaseFields($entity_type, $entity, array $base_fields) {
```

Comment:
Why remove the `\stdClass` type hint? This is here to clarify that this is not a real entity but rather a "mocked" value object. It is literally a `\stdClass` object so the type hint is correct. Am I missing something?

Reply:
This, as well as other changes, is a result of Rector's and drupal-check report. I don't remember what static analysis rule this was violating specifically, but it would not pass those tools.

We can revert and add an ignore comment for those tools to ignore this. 

Please advise